packages/fluentui/docs/src/examples/components/Pill/Types/PillExampleDefault.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import { Pill } from '@fluentui/react-northstar';
+
+const ExamplePill = () => <Pill>This is a default Pill</Pill>;
+
+export default ExamplePill;
```

Comment:
```suggestion
export default PillExample;
```

Please also rename a filename

Reply:
done